BankingManagement/account/account.go

```diff
@@ -0,0 +1,68 @@
+package account
+
+import (
+	"fmt"
+	history "hello/Assignment/github.com/BankingManagement/TransactionHistory"
+	"time"
+
+	uuid "github.com/google/uuid"
+)
+
+type accInterface interface {
+	Create(id string, name string) bool
+	Withdraw(amount int) bool
+	Deposit(amount int) bool
+	History()
+	Delete()
+}
+type Account struct {
+	Name, Id    string
+	money       int
+	Trandetails []history.Transaction
+}
+
+func (acc *Account) Create(id string, name string) bool {
+	acc.Name = name
+	acc.Id = id
+	acc.money = 0
+	return true
+}
+func (acc *Account) Withdraw(amount int) bool {
+
+	if amount > acc.money {
+		return false
+	} else {
+		acc.money = acc.money - amount
+
+		var new history.Transaction
+		new.Id = uuid.NewString()
+		new.Amount = amount
+		new.TimeStamp = time.Now().Format("Monday, 02-Jan-06 15:04:05 MST")
+		new.Type = "credit"
+
+		acc.Trandetails = append(acc.Trandetails, new)
+		return true
+	}
+}
+func (acc *Account) Deposit(amount int) bool {
```

Comment:
As you can see most of the code block in Deposite and Withdraw method is the same, so you can pull of common part in unexported method or function and call that function from both of these methods, In this way, we can reuse the code and code maintenance will get reduce